src/service/db/organization.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 Zinc Labs Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::common::{
+    infra::{config::ORGANIZATION_SETTING, db as infra_db},
+    meta::organization::OrganizationSetting,
+    utils::json,
+};
+use bytes::Bytes;
+
+pub async fn set_org_setting(
+    org_name: &str,
+    setting: &OrganizationSetting,
+) -> Result<(), anyhow::Error> {
+    let db = &infra_db::DEFAULT;
+    let key = format!("/organization/{}", org_name);
```

Comment:
I suggest giving it a second path like `/organization/setting/{}`, because if we have another metadata for `organization` the prefix will be a problem.